src/Ajax/UpdateCodeMirrorCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: dpino
+ * Date: 10/1/19
+ * Time: 11:30 AM
+ */
+
+namespace Drupal\strawberryfield\Ajax;
+use Drupal\Core\Ajax\CommandInterface;
+class UpdateCodeMirrorCommand implements CommandInterface
+{
```

Comment:
Move bracket to the same line as the class

Reply:
Resolved by e5c480833d0a9dde6f6e6385b60a4a363516ca1c